shell/platform/windows/flutter_window.cc

```diff
@@ -327,8 +320,8 @@ bool FlutterWindow::OnBitmapSurfaceUpdated(const void* allocation,
   bmi.bmiHeader.biBitCount = 32;
   bmi.bmiHeader.biCompression = BI_RGB;
   bmi.bmiHeader.biSizeImage = 0;
-  int ret = ::SetDIBitsToDevice(dc, 0, 0, row_bytes / 4, height, 0, 0, 0,
-                                height, allocation, &bmi, DIB_RGB_COLORS);
+  int ret = SetDIBitsToDevice(dc, 0, 0, row_bytes / 4, height, 0, 0, 0, height,
```

Comment:
Is this a necessary part?

Reply:
This isn't strictly necessary but for bookkeeping purposes I'd prefer to fully revert https://github.com/flutter/engine/pull/49262. This will make the reland easier to review as there'll be fewer changes between the original PR and the reland.